packages/odyssey-react/src/components/Tag/Tag.module.scss

```diff
@@ -12,20 +12,20 @@
 
 .list {
   display: inline-block;
-  margin-block-end: -$spacing-xs;
-  margin-inline-start: $spacing-xs;
+  margin-block-end: calc(-1 * var(--ListSpaceMarginBlockEnd));
+  margin-inline-start: var(--ListSpaceMarginInlineStart);
   list-style-type: none;
 }
 
 .tag {
-  @include border-radius($base-border-radius);
+  @include border-radius(var(--TagBorderRadius));
 
   display: inline-block;
-  margin-block-start: 0;
-  margin-block-end: $spacing-xs;
-  margin-inline-start: 0;
-  margin-inline-end: $spacing-xs;
-  padding-block: 0;
-  padding-inline: $spacing-xs-em;
```

Comment:
For the `em` version, would this be the correct replacement: `-$spacing-xs-em` -> `SpaceEmXs`

Reply:
yes!